src/test/java/xyz/vegaone/easytracking/userstory/UserStoryServiceTest.java

```diff
@@ -68,16 +75,8 @@ public void createUserStoryTest(){
     @Test
     public void getUserStoryTest(){
         //given
-        UserStory userStory = new UserStory();
-        userStory.setTitle(TITLE);
-        userStory.setDescription(DESCRIPTION);
-        userStory.setOwner(OWNER);
-        userStory.setEstimation(ESTIMATION);
-        userStory.setPriority(PRIORITY);
-        userStory.setStatus(STATUS);
-        userStory.setProjectId(PROJECT_ID);
 
-        UserStory savedUserStory = userStoryService.createUserStory(userStory);
+        UserStory savedUserStory = userStoryService.createUserStory(buildAndSaveUserStory());
 
         Task task = new Task();
```

Comment:
The task can be extracted as well into a separate private method. See my comments in the [other PR](https://github.com/sabaubogdan/easytracking/pull/9#discussion_r160050490).